packages/analytics-core/test/event-bridge/event-bridge-container.test.ts

```diff
@@ -0,0 +1,39 @@
+import { EventBridgeContainer } from '../../src/event-bridge/event-bridge-container';
+import * as EventBridgeModule from '../../src/event-bridge/event-bridge';
+
+describe('EventBridgeContainer', () => {
+  beforeEach(() => {
+    EventBridgeContainer.instances = {};
+  });
+
+  describe('getInstance', () => {
+    test('should create new instance', () => {
```

Comment:
Feels this test has already been included in the next test case. Might be we can merge those two?